yarn.config.cjs

```diff
@@ -577,8 +572,7 @@ function expectCorrectWorkspaceChangelogScripts(workspace) {
 /**
  * Expect that if the workspace package lists another workspace package within
  * `dependencies` or `devDependencies`, the version used within the dependency
- * range is exactly equal to the current version of the dependency (and the
- * range uses the `^` modifier).
+ * is `workspace:^`.
```

Comment:
An update that enforces all workspace package dependencies to use the workspace version. This is automatically updated using `yarn publish` and aligns with how Snaps approaches workspace package dependencies: https://github.com/MetaMask/snaps/blob/main/yarn.config.cjs#L650-L672